fleeto_backend/config/routes.rb

```diff
@@ -1,3 +1,22 @@
 Rails.application.routes.draw do
+
+  devise_for :customers
+
+  api_version(:module => "V1", :path => {:value => "v1"}) do
```

Comment:
Would it make more sense if the versioning was on 'customer' and 'driver' level?
So instead of "v1/customer" and "v1/client", it'll be "customer/v1" and "client/v1".

I'm just asking...